src/common/sass/widgets/_panel.scss

```diff
@@ -52,7 +52,9 @@
     &:overview, 
     &:focus, 
     &:checked {
+      margin-top: 2px; // Needed to keep button from jumping up.
```

Comment:
I think this is breaking things. Clicking the top of the panel doesn't open popups anymore, and clicking the top of the Activities button when the overview is open doesn't close it.